frontend/src/pages/CourseSelector/courseMenu/CourseMenu.jsx

```diff
@@ -174,25 +178,43 @@ export default function CourseMenu() {
   );
 }
 
-const MenuItem = ({ selected, courseCode, activeCourse, setActiveCourse }) => {
+const MenuItem = ({ selected, courseCode, activeCourse, setActiveCourse, accurate }) => {
   const dispatch = useDispatch();
   const handleClick = () => {
     dispatch(courseTabActions("ADD_TAB", courseCode));
     setActiveCourse(courseCode);
   };
 
+  const renderAccurateNote = () => {
+    if (!accurate){
+      return (<WarningIcon text="This course info may be inaccurate" />);
+    }
+  }
+
   return (
     <Menu.Item
       className={`text menuItemText ${selected !== undefined && "bold"} 
       ${activeCourse === courseCode && "activeCourse"}`}
       key={courseCode}
       onClick={handleClick}
     >
-      {courseCode}
+      {courseCode} {renderAccurateNote()}
```

Comment:
I think it's better to avoid the absolute positioning you do for the warning icon (in line 212) since it can be a bit unreliable. 
Instead, you can wrap these two into separate span elements and put a flex div container around it. 

```less
.menuItem {
  display: flex;
  align-items: center;
  gap: 4px;
}
```